### PR TITLE
Add voltage levels to the request when geting equipment infos

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 1]
```diff
@@ -494,9 +494,9 @@ public CreatedStudyBasicInfos insertStudy(UUID studyUuid, String userId, Network
         Objects.requireNonNull(importParameters);
 
         StudyEntity studyEntity = self.saveStudyThenCreateBasicTree(new StudyEntity(studyUuid, networkUuid, networkId, caseFormat, caseUuid, caseName,
-                    null, null, null, defaultNonEvacuatedEnergyProvider, defaultDynamicSimulationProvider,
-                    loadFlowParametersUuid, shortCircuitParametersUuid, dynamicSimulationParametersEntity, voltageInitParametersUuid, securityAnalysisParametersUuid,
-                    sensitivityAnalysisParametersUuid, null, importParameters, StudyIndexationStatus.INDEXED, new StudyVoltageInitParametersEntity()),
+                        null, null, null, defaultNonEvacuatedEnergyProvider, defaultDynamicSimulationProvider,
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:497]: remove useless tabulation ?

File: src/test/java/org/gridsuite/study/server/NetworkMapTest.java [hunk 2]
```diff
@@ -539,18 +544,26 @@ private MvcResult getNetworkElementsIds(UUID studyUuid, UUID rootNodeUuid, Strin
...
+    private MvcResult getNetworkElementsInfos(UUID studyUuid, UUID rootNodeUuid, String infoType, String elementType, List<Double> nominalVoltages, String requestBody, String responseBody) {
+        UUID stubUuid = wireMockUtils.stubNetworkElementsInfosPost(NETWORK_UUID_STRING, infoType, elementType, nominalVoltages, responseBody);
 
         MockHttpServletRequestBuilder mockHttpServletRequestBuilder = post("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network/elements", studyUuid, rootNodeUuid)
                 .queryParam(QUERY_PARAM_INFO_TYPE, infoType)
-                .queryParam(String.format(QUERY_FORMAT_OPTIONAL_PARAMS, QUERY_PARAM_DC_POWERFACTOR), Double.toString(LoadFlowParameters.DEFAULT_DC_POWER_FACTOR))
+                .queryParam(QUERY_PARAM_ELEMENT_TYPE, elementType)
+                .queryParam(String.format(QUERY_FORMAT_OPTIONAL_PARAMS, QUERY_PARAM_DC_POWERFACTOR), Double.toString(LoadFlowParameters.DEFAULT_DC_POWER_FACTOR));
+        if (nominalVoltages != null) {
+            for (Double voltage : nominalVoltages) {
```
Comment (sBouzols) [inline src/test/java/org/gridsuite/study/server/NetworkMapTest.java:555]: forEach ?

**Review by YenguiSeddik — APPROVED** [review]

File: src/test/java/org/gridsuite/study/server/NetworkMapTest.java (hunk 2 above)
Comment (AAJELLAL) [reply src/test/java/org/gridsuite/study/server/NetworkMapTest.java]: Done.

**Review by sBouzols — APPROVED** [review]
Code Review OK
Test OK 
alea creation from map for lines with multiple conf for nominalVoltages filter
tested with no nominalvoltage selection, all, some combination

**Review by AAJELLAL — APPROVED** [review]
Code and tests are OK.

**Pull request merged**